SimulationAgent/Simulation.cs

```diff
@@ -87,6 +89,45 @@ public void Stop()
             this.runner.Stop();
         }
 
+        /// <summary>
+        /// Delete a list of devices
+        /// </summary>
+        public async Task DeleteDevices(List<string> ids)
+        {
+            this.log.Info("Update simulation", () => { });
+            var simulation = (await this.simulations.GetListAsync()).FirstOrDefault();
```

Comment:
why only use the first simulation by default?

Reply:
Currently only one simulation is supported in the service.

Reply:
maybe just add a comment here. LGTM.